config/locales/en.yml

```
@@ -85,22 +85,24 @@ en:
               <li>what you click on while you’re visiting the site</li>
               <li>the device and browser you’re using</li>
             </ul>
-            We don’t collect or store your personal information (for example your name or address) so this information can’t be used to identify who you are.
-            We don’t allow Google to use or share our analytics data.
+            We don’t collect or store your personal information (for example your name or address) so this information can’t be used to identify who you are.<br><br>
```

Best to remove the break tags and replace with paragraphs:
```suggestion
            <p class="govuk-body">We don’t collect or store your personal information (for example your name or address) so this information can’t be used to identify who you are.</p>
```